library/cpp/transport/TFDTransport.cpp

```diff
@@ -28,7 +28,7 @@
 
 #ifdef _WIN32
 #include <io.h>
-#include <winutils.h>
+typedef ptrdiff_t ssize_t;
```

Comment:
Couldn't we use `ptrdiff_t` from `cstddef` header file?

Reply:
Here you go: http://en.cppreference.com/w/cpp/language/typedef